raxx_cowboy/lib/raxx/cowboy/handler.ex

```diff
@@ -59,10 +60,15 @@ defmodule Raxx.Cowboy.Handler do
     {:ok, body, _req} = parse_req_body(req, content_type)
 
     # {peer, req} = :cowboy_req.peer req
+    
+    scheme = case transport do
+      :ssl -> "https"
```

Comment:
It still seams strange we match on http earlier and set it to https here? When setting up ssl did you use cowboy.start_https instead of start_http?